src/test/java/com/skynest/BaseTest.java

```diff
@@ -17,4 +25,29 @@ public void setupEnvironment() throws URISyntaxException {
         skyNestBackendClient = new SkyNestBackendClient(targetDomain);
     }
 
+    public void loginUser(String email, String password) {
+        LoginRequest body = new LoginRequest(email, password);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_OK);
+    }
+
+    public void loginAs(Roles role) {
+        switch (role) {
+            case ADMIN:
+                loginUser("christinezahar@yahoo.com", "Hris8833");
+                break;
+            case WORKER:
+                loginUser("hristina.zaharieva@htecgroup.com", "Hris8833");
+                break;
+        }
+    }
+
+    public UUID getLoggedUserId() throws IOException {
+        Response response = skyNestBackendClient.getLoggedUser();
+        LoggedUserResponse loggedUserResponse = JsonTransformer.mapResponse(response, LoggedUserResponse.class);
```

Comment:
why not using RestAssured embedded method `as` to achieve this?

`response.as(LoggedUserResponse.class)`